src/main/java/build/buildfarm/common/redis/RedisPriorityQueue.java

```diff
@@ -122,16 +122,17 @@ public boolean removeAll(JedisCluster jedis, String val) {
    */
   @Override
   public String dequeue(JedisCluster jedis, int timeout_s) throws InterruptedException {
+    double pollInterval = 0.1;
```

Comment:
Please make this into a parameter for the PriorityQueue object and inject it via a config.